torch/csrc/cudnn/Handles.cpp

```diff
@@ -39,17 +39,7 @@ cudnnHandle_t getCudnnHandle()
 
 void useCurrentStream(cudnnHandle_t handle, THCState *state)
 {
-  cudnnStatus_t status = cudnnSetStream(
-      handle,
-      THCState_getCurrentStream(state));
-  if (status == CUDNN_STATUS_BAD_PARAM) {
-    std::cerr << "WARNING: invalid handle\n";
-  } else if (status == CUDNN_STATUS_MAPPING_ERROR) {
-    std::cerr <<
-        "WARNING: Mismatch between user stream and the cuDNN handle context\n";
-  } else if (status != CUDNN_STATUS_SUCCESS) {
-    std::cerr << "WARNING: Could not set cuDNN to use current stream\n";
-  }
+  CHECK(cudnnSetStream(handle, THCState_getCurrentStream(state)));
```

Comment:
now you can get rid of this one-liner function `useCurrentStream` altogether :)

Reply:
just replicate `CHECK(cudnnSetStream(handle, THCState_getCurrentStream(state)));` everywhere maybe.
It seems cleaner, but if someone things otherwise (i.e. keeping useCurrentStream), i'm fine with it.

Reply:
sounds good. Fixing that here: https://github.com/pytorch/pytorch/pull/2990